lib/segment/src/segment.rs

```diff
@@ -1984,15 +1990,15 @@ mod tests {
         segment.delete_point(103, 4.into()).unwrap();
         let segment_info = segment.info();
         assert_eq!(segment_info.num_points, 1);
-        assert_eq!(segment_info.num_vectors, 1);
+        assert_eq!(segment_info.num_vectors, 2); // We don't propagate deletes to vectors at this time
 
         // Delete vector of point 6, vector count should now be zero
         segment
             .delete_vector(104, 6.into(), DEFAULT_VECTOR_NAME)
             .unwrap();
         let segment_info = segment.info();
         assert_eq!(segment_info.num_points, 1);
-        assert_eq!(segment_info.num_vectors, 0);
+        assert_eq!(segment_info.num_vectors, 1);
```

Comment:
Doesn't this new behavior make the intention of this test a little different? At least in terms of what the comments still say.

Reply:
Good point, thanks for catching this. I'll comment it out, as I think it'll still be useful if we ever get this behavior of propagating point deletions to vectors back.

Reply:
I agree, to complement this I think it would be helpful to have an issue so we don't forget it, if we haven't already :)

Reply:
Good idea. I'll create an issue for it.